src/igvfd/search_views.py

```diff
@@ -117,7 +117,7 @@ def report(context, request):
             ),
             IDResponseField(),
             ContextResponseField(),
-            BasicReportWithFacetsResponseField(
+            MultipleTypesReportWithFacetsResponseField(
```

Comment:
Make a new report endpoint instead of changing this one. Something like `multitype-report` or `multireport`?

Reply:
Sounds good, will do.